CHANGELOG.md

```diff
@@ -126,6 +126,8 @@ New modules
 Additions to existing modules
 -----------------------------
 
+* In `Algebra.Properties.Semigroup` adding consequences for associativity for semigroups
+
```

Comment:
Need to insert here all the new names, once we have (eventually) agree them...
```suggestion
```agda
  uv≈w⇒xu∙v≈xw : ∀ x → (x ∙ u) ∙ v ≈ x ∙ w
```

etc.